src/servers/graphql-server.ts

```diff
@@ -82,7 +82,7 @@ export const startApolloServer = async ({
       const apiKey = context.req?.apiKey ?? null
       // @ts-expect-error: TODO
       const apiSecret = context.req?.apiSecret ?? null
-      const uid = token?.uid ?? null
+      const userId = (token?.uid as UserId) ?? null
```

Comment:
Type casting to domain types should happen within the app layer to guarantee application of any needed preconditions / validation.